### ISSUE TITLE
Support for booster vaccinations?

### ISSUE THREAD

**Issue by jbx1** [eu-digital-green-certificates/dgc-testdata#441]
## Issue Description
Schema version 1.3.0 (9th June 2021), which is still the latest release, ​states that a certificate will only contain one vaccination entry:

> "Vaccination group, if present, MUST contain exactly 1 (one) entry describing exactly one vaccination event."

How will booster (3rd) jabs, from different manufacturers than the first/second one be represented? Will there be a different Vaccination Certificate altogether, or will this schema be enhanced to support multiple entries (as it was originally before 1.3.0)?

For example: 1st and 2nd jabs were AstraZeneca, but Booster (3rd) jab is Pfizer. 

## Proposed Solution

Specifications clarification needed and test-data needs to be updated with new examples that demonstrate how boosters will be included.

**Comment by Lorenzo-Venturelli** [comment]
The certificate represents exactly one event (in this case, exactly one jab). The booster will be represented by increasing the number of administered jabs (for example 3/2 for two doses vaccines or 2/1 for single dose vaccines). This is because the "required doses" field is only about the "main vaccination cycle" while boosters are potentially unlimited.

Regarding the mixed vaccination, many countries are already proposing mixed vaccines even for the main cycle. This is not a problem as the EU only use the most recent certificate to evaluate the person's status so there is no need to know how and when the previous doses have been administered.

In conclusion, if a country decides to check the whole vaccination history (for example to exclude natural immunity, like the UK), the person needs to present multiple certificates, one for each relevant event.

**Comment by merlinschumacher** [comment]
For the few booster certificates I've seen from Germany, which have been issued by pharmacies, the counter isn't entered correctly. I've received a booster today and the first two jabs are listed as 1/2, 2/2, while the booster jab is listed as 3/3. Some people who got J&J now have 1/1 and 2/2.
The spec isn't enforced properly.

**Comment by grambas** [comment]
also just saw booster vaccine certificate listed as 3/3 (Lithuania)

**Comment by Lorenzo-Venturelli** [comment]
I must review my previous post.
Since mid-November, in Italy as well the booster shot is marked as 3/3 (or 2/2, depending on the case).
